sys/include/xtimer/implementation.h

```diff
@@ -64,7 +64,6 @@ static inline uint32_t _xtimer_lltimer_mask(uint32_t val)
 uint64_t _xtimer_now64(void);
 int _xtimer_set_absolute(xtimer_t *timer, uint32_t target);
 void _xtimer_set64(xtimer_t *timer, uint32_t offset, uint32_t long_offset);
-void _xtimer_set(xtimer_t *timer, uint32_t offset);
```

Comment:
nit picky: why delete this one, I'd rather delete the duplicate below in L73 to have `_xtimer_set` and `_xtimer_set64` together like (and may in reversed order) like all other internal calls with 32 and 64 variants.

Reply:
will do.